sdk/translation/Azure.AI.Translation.Document/tests/SessionRecords/DocumentTranslationClientLiveTests/GetAllDocumentStatusesFilterByIdsTest.json

```diff
@@ -479,164 +644,176 @@
+          "ARRAffinitySameSite=71659a59bc7ec5e68f754ab1110af7204cc0a6ca9121108e19aeec14032c3bc4;Path=/;HttpOnly;SameSite=None;Secure;Domain=mtbatch.nam.microsofttranslator.com"
         ],
         "Strict-Transport-Security": "max-age=31536000; includeSubDomains; preload",
         "Transfer-Encoding": "chunked",
-        "X-Content-Type-Options": "nosniff",
+        "x-content-type-options": "nosniff",
         "X-Powered-By": "ASP.NET",
-        "X-RequestId": "abf45351-4edb-47e1-8092-8be03d86cdb8"
+        "X-RequestId": "ae7e10cf-af77-4c48-af70-5e23f321d002"
       },
       "ResponseBody": {
         "value": [
           {
-            "path": "https://storagedoctranslator.blob.core.windows.net/target1402114609/File_1.txt",
-            "sourcePath": "https://storagedoctranslator.blob.core.windows.net/source1526944569/File_1.txt",
-            "createdDateTimeUtc": "2021-08-04T20:12:41.8278051Z",
-            "lastActionDateTimeUtc": "2021-08-04T20:12:45.6152015Z",
+            "path": "https://documentstorageleithy.blob.core.windows.net/target1402114609/File_1.txt",
+            "sourcePath": "https://documentstorageleithy.blob.core.windows.net/source1526944569/File_1.txt",
+            "createdDateTimeUtc": "2021-08-18T18:07:10.1354597Z",
+            "lastActionDateTimeUtc": "2021-08-18T18:07:13.4730033Z",
             "status": "Succeeded",
             "to": "fr",
             "progress": 1,
-            "id": "000a52ef-0000-0000-0000-000000000000",
+            "id": "000ae8eb-0000-0000-0000-000000000000",
             "characterCharged": 16
           },
           {
-            "path": "https://storagedoctranslator.blob.core.windows.net/target1402114609/File_0.txt",
-            "sourcePath": "https://storagedoctranslator.blob.core.windows.net/source1526944569/File_0.txt",
-            "createdDateTimeUtc": "2021-08-04T20:12:41.5511869Z",
-            "lastActionDateTimeUtc": "2021-08-04T20:12:45.7174345Z",
+            "path": "https://documentstorageleithy.blob.core.windows.net/target1402114609/File_0.txt",
+            "sourcePath": "https://documentstorageleithy.blob.core.windows.net/source1526944569/File_0.txt",
+            "createdDateTimeUtc": "2021-08-18T18:07:10.1294254Z",
+            "lastActionDateTimeUtc": "2021-08-18T18:07:13.4246116Z",
             "status": "Succeeded",
             "to": "fr",
             "progress": 1,
-            "id": "000a52ee-0000-0000-0000-000000000000",
+            "id": "000ae8ea-0000-0000-0000-000000000000",
             "characterCharged": 16
           }
         ]
       }
     },
     {
-      "RequestUri": "https://r-doctranslator.cognitiveservices.azure.com/translator/text/batch/v1.0/batches/95d1a6fa-cbcc-4eda-b8c5-7f9dea903155/documents?ids=000a52ef-0000-0000-0000-000000000000\u0026statuses=\u0026createdDateTimeUtcStart=0001-01-01T00%3A00%3A00.0000000Z\u0026createdDateTimeUtcEnd=0001-01-01T00%3A00%3A00.0000000Z\u0026$orderBy=",
```

Comment:
Looks like this was a bug. We are going to have to report it in the changelog.
Could you please add it to the `Bugs Fixed` section https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md#bugs-fixed ?
You could say something like:
```
- `GetDocumentStatuses` and `GetDocumentStatusAsync` no longer set `CreatedAfter`  and `CreatedBefore` if the user doesn't set it.
```